lib/check_params.h

```diff
@@ -821,6 +825,23 @@ void printQudaMultigridParam(QudaMultigridParam *param) {
     P(setup_ca_lambda_max[i], INVALID_DOUBLE);
 #endif
 
+#ifdef INIT_PARAM
+    P(filter_startup_vectors[i], 1);
+    P(filter_startup_iterations[i], 1000);
+    P(filter_startup_rescale_frequency[i], 50);
+    P(filter_iterations_between_vectors[i], 150);
+    P(filter_lambda_min[i], 1.0);
```

Comment:
Again, just wondering why 1.0 is a good default for a min here?

Reply:
Semi-empirical, in some cases the gapping relative to, say, 1e-2, was necessary to get good enhancement of very small eigenvalues; in other cases it was necessary to get enhancement when I was testing far from the critical point (aka when I didn't bother finding the critical point in the Wilson case).